### ISSUE TITLE
CVE-2022-23812: YOUR CODE IS INFECTED WITH MALICIOUS DEPENDENCY - node-ipc 

### ISSUE THREAD

**Issue by lgg** [imba/imba#659]
Newest version of node-ipc delete all users's files from device. You should not use this dependency anymore!

You can learn more here: https://gist.github.com/MidSpike/f7ae3457420af78a54b38a31cc0c809c

Check possible solution that already applied in vue.js: https://github.com/vuejs/vue-cli/issues/7054#issuecomment-1068677029

also check more here: https://snyk.io/blog/peacenotwar-malicious-npm-node-ipc-package-vulnerability/

**Comment by codeluggage** [comment]
We're at `9.1.4` now. Is this still the recommended way to mitigate this? 

```json
  "overrides": {
    "node-ipc@>9.2.1 <10": "9.2.1",
    "node-ipc@>10.1.0": "10.1.0"
  }
```

**Comment by somebee** [comment]
Is this not fixed by forcing the exact `9.1.4` version? No longer getting an dependabot alerts.
